Earthfile

```diff
@@ -495,8 +495,10 @@ uki-artifacts:
     FROM +base-image --BUILD_INITRD=false
     RUN /usr/bin/immucore version
     RUN ln -s /usr/bin/immucore /init
+    RUN mkdir -p /oem # be able to mount oem under here if found
```

Comment:
this makes it easier to deal with mounts later

Reply:
why does this only apply for UKI artifacts?

Reply:
there doesnt seem to be an oem dir on the rootfs when generating the uki artifact so if we want to mount something in there on boot, the path needs to exist before. No ieda why the rootfs doesnt have oem....

Reply:
but should we create it at the Dockerfile then?

Reply:
No idea, I need to investigate because the isos and images DO have it...so it has to be created somewhere...